src/main/webapp/util/Validator.js

```diff
@@ -1,11 +1,20 @@
+import * as _ from 'lodash';
 import { PREDEFINED_VALIDATORS_DICT } from '../constants/preDefinedRegex';
 import { getFormattedMessage } from './messageUtil';
-import {
-    parseNumberValidator,
-    parseRegexRawStr,
-    parseStringValidator,
-    parseFunctionRawStr,
-} from './uccConfigurationValidators';
+
+const parseFunctionRawStr = (rawStr) => {
+    let err;
+    let result;
+
+    try {
+        // eslint-disable-next-line no-eval
+        result = eval(`(${rawStr})`);
```

Comment:
Do you want to drop this check in this PR or later?

Reply:
Looking into it, I am wondering if we remove this, we will not be able to execute the function on the fly when the user submits the form.  (If [saveValidators](https://splunk.github.io/addonfactory-ucc-generator/advanced/save_validator/) is present in the global config file then only)

Reply:
Let's have a separate PR for this removal.